defaults/main.yml

```diff
@@ -1,7 +1,32 @@
+# note: the booleans are quoted otherwise they are rendered capitalised.
 ---
+buildkite_agent_major_version: 3
 buildkite_agent_conf_dir: "/etc/buildkite-agent"
 buildkite_agent_token: ""
-buildkite_agent_name: "buildkite-agent"
+buildkite_agent_debug: "false"
+buildkite_agent_enabled: true
+# if you're building a golden image and don't want it to connect to
+# buildkite during build ptocess, set this to 'stopped'.
```

Comment:
I tried testing this behavior as part of https://github.com/azavea/ansible-buildkite-agent/pull/2, but was unable to reproduce it.

Reply:
I think I'll remove this entirely because of how I've (since) decided to deploy changes to the agent cluster (see https://github.com/azavea/ansible-buildkite-agent/pull/3#discussion_r181770528).

Reply:
Ah, it was also only in this branch because of a bad merge on my part. Apologies.